x-pack/plugins/cross_cluster_replication/public/app/services/follower_index_default_settings.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+const defaultSettings = {
+  maxReadRequestOperationCount: 5120,
+  maxOutstandingReadRequests: 12,
+  maxReadRequestSize: '32mb',
+  maxWriteRequestOperationCount: 5120,
+  maxWriteRequestSize: '9223372036854775807b',
+  maxOutstandingWriteRequests: 9,
+  maxWriteBufferCount: 2147483647,
+  maxWriteBufferSize: '512mb',
+  maxRetryDelay: '500ms',
+  readPollTimeout: '1m',
+};
+
+export const getSettingDefault = (name) => {
+  if(!defaultSettings[name]) {
+    throw new Error(`Unknown setting ${name}`);
```

Comment:
This string will probably be shown to the user eventually, e.g. within the fatal error screen, so I think we need to internationalize it.

Reply:
to me, `throw new Error()` is for internal code usage, any messages meant for user consumption from thrown errors should be displayed via `catch()` blocks. I also don't think the fatal error screen necessarily needs to be internationalized because it's meant for last-resort debugging/stack trace